exercises/Ex5Christmas.js

```diff
@@ -1,22 +1,13 @@
 function nextChristmas(){ 
     var todaysDate = new Date();
-    if(todaysDate.getMonth() == 0){
-        if(todaysDate.getDate() == 7){
-            alert("Merry Christmas!!!");
-        }
-        else if(todaysDate.getDate() == 6){
-            alert("Tomorrow is Christmas!!!");
-        }
-        else if(todaysDate.getDate() < 6){
-            var daysForChristmas = 7 - todaysDate.getDate();
-            alert("Christmas is coming. There are " + daysForChristmas + " days left for Chrismas.");
-        }
-    }
-    else{
-        var currYear = todaysDate.getFullYear()+1;
-        var ChristmasDate = new Date(currYear, 0, 07);
-        var daysForChristmas = Math.ceil((ChristmasDate.getTime() - todaysDate.getTime())/(24*3600*1000)); 
-        alert("There are " + daysForChristmas + " days left for Chrismas.");
+    var currYear = todaysDate.getFullYear();
+    //var ChristmasDate = new Date(currYear, 0, 07);
+    var ChristmasDate = new Date(currYear, 0, 07);
+    if(todaysDate >= ChristmasDate){
+        //var currYear = todaysDate.getFullYear()+1;
+        var ChristmasDate = new Date(currYear+1, 0, 07);
     }
+    return Math.ceil((ChristmasDate-todaysDate)/3600/24/1000);
 }
-nextChristmas();
\ No newline at end of file
+var result = nextChristmas();
```

Comment:
It would be nicer to code the function to work with any Date passed in as a parameter.
Then you can call it for Today from main program